logfire/_internal/cli.py

```diff
@@ -398,7 +435,9 @@ def _main(args: list[str] | None = None) -> None:
 
     parser.add_argument('--version', action='store_true', help='show the version and exit')
     global_opts = parser.add_argument_group(title='global options')
-    global_opts.add_argument('--logfire-url', default=LOGFIRE_BASE_URL, help=argparse.SUPPRESS)
+    url_or_region_grp = global_opts.add_mutually_exclusive_group()
+    url_or_region_grp.add_argument('--logfire-url', help=argparse.SUPPRESS)
+    url_or_region_grp.add_argument('--region', choices=REGIONS, help='the region to use')
```

Comment:
`logfire auth -h` doesn't show `--region`

Reply:
Can we not improve this?

Reply:
anyway i guess we leave it for another PR

Reply:
> Can we not improve this?

I can relatively easily move the two mutually exclusive `--logfire-url` and `--region` options for each command.